libstb/secvar/backend/backend-common.c

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: Apache-2.0
+/* Copyright 2019 IBM Corp. */
+#include <opal.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+#include <stdint.h>
+#include <device.h>
+#include <ccan/endian/endian.h>
+#include <mbedtls/error.h>
+#include "backend-common.h"
+#include "../secvar.h"
+
+bool setup_mode;
+
+int add_to_variable_bank(struct secvar *secvar, const char *data, uint64_t dsize)
```

Comment:
This should probably be renamed, and definitely commented. The function updates an existing variable's data field, it doesn't "add" anything to the bank.

Should probably be something more like `update_variable_in_bank`, or something similar.

Reply:
The comments are given in .h file where it is declared.